apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -564,6 +564,37 @@ type Storage struct {
 // +kubebuilder:validation:Maximum=4094
 type VLANID int32
 
+type LLDPProcessed struct {
+	// https://github.com/openstack/python-ironic-inspector-client/blob/master/ironic_inspector_client/resource.py#L24-L59
+	Interface                         string           `json:"interface,omitempty"`
+	Mac                               string           `json:"mac,omitempty"`
+	NodeIdent                         string           `json:"node_ident,omitempty"`
+	SwitchCapabilitiesEnabled         []string         `json:"switch_capabilities_enabled,omitempty"`
```

Comment:
I wonder if we should create a Switch substructure just to make the API a bit more usable.